app/javascript/components/Booking.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect, useState} from 'react';
+import RoomsFilter from "./RoomsFilter";
+import RoomsList from "./RoomsList";
+import axios from "axios";
+
+const Booking = ({defaultOffice}) => {
+    const host = 'http://127.0.0.1:3000'
```

Comment:
Create specific service to cover any calls to servers. Don't use Axios directly. 

Do something like this

const get = (url, body, token) => {
   call  axios.get(...) here and then import this get function to files where you need to use a get request 
}

import useApi "./../useAPi"

const { get } = useApi()

try {
 const response = await get(`${host}/offices`)
} catch {
  console.error("handle errors")
}